dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -47,6 +47,9 @@ using namespace Microsoft::Windows::AppNotifications::Helpers;
 
 namespace winrt::Microsoft::Windows::AppNotifications::implementation
 {
+    // Must be static for AppNotificationManager::Deserialize
+    static wil::unique_event s_waitHandleForArgs;
```

Comment:
Why static? Can you make it a member variable?

Reply:
Other Deserialize functions like LaunchActivatedEventArgs, FileActivatedEvent args use the static keyword with Deserialize. The ExtensionMap in ExtensionContract.h is expecting a winrt::Windows::Foundation::IInspectable(*factory)(winrt::Windows::Foundation::Uri const& uri). I guess by using the & operator on a static function you get the factory. Compile errors occur when removing the static tag.

Reply:
Can you do this and get rid of the static in the mapping? 
&winrt::Microsoft::Windows::AppNotifications::implementation::AppNotificationManager::GetDefault()::Deserialize

Reply:
I guess you answered the question : )

Reply:
@aeloros We would like to reference a member variable in our Deserialize function but we are having issues since it can't be static. What should we do in this scenario? Is there a limitation with Deserialize that can only be a static function? My guess is that the & operator returns the factory for a static function.

If you can help us get to winrt::Windows::Foundation::IInspectable(*factory)(winrt::Windows::Foundation::Uri const& uri), that would be great :)

Reply:
Is this a blocker for checking in? It seems like we can afford to address this as a bug in the big PR.

Reply:
I talked to @pmpurifoy . I believe we have a solution that will work with these limitations. He will give it a shot and reply back.